### ISSUE TITLE
Examples in the wild

### ISSUE THREAD

**Issue by tshort** [JuliaDynamics/DrWatson.jl#195]
It'd be nice to point to repositories using DrWatson for real work. I imagine most are in private repos, but I assume that some are/will be public.

My best attempt at a GitHub search only shows one:

https://github.com/search?q=drwatson+filename%3AProject.toml+language%3AJulia&type=Repositories

**Comment by Datseris** [comment]
Good idea, let's collect a few here before adding them to the docs. I have this one: https://github.com/Datseris/arXiv_1904.05108

**Comment by Datseris** [comment]
(and as you said, I have three more in private repos that will become public after publication)

**Comment by treigerm** [comment]
There is also https://github.com/cambridge-mlg/Covid19.

**Comment by TheCedarPrince** [comment]
I have one here @Datseris: https://github.com/TheCedarPrince/NeuriViz
